containers/elabdev/clean.sh

```
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+docker rmi elabftw-elabimg
+docker rmi elabftw/elabdev:edge
+set -eu
+# sudo because mysql files will be root owned
+sudo rm -rv ${HOME}/.local/share/elabdev
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)